docs/source/user/aerodyn/driver.rst

```diff
@@ -0,0 +1,497 @@
+The motion of the base may be: fixed (`BaseMotionType=0`), sinusoidal (`BaseMotionType=1`) or arbitrary (`BaseMotionType=2`). 
+The turbine base motion is applied at each time step before applying the initial position and orientation of the turbine base.
+A sinusoidal motion implies that one degree of freedom (`DegreeOfFreedom`) of the turbine base is moving according to a sine function of a given amplitude (`Amplitude`) and frequency (`Frequency`), with zero phase.
+The 6 possible degrees of freedom correspond to translations or rotations of the base frame in global coordinates (g) (e.g. surge, sway, heave, roll, pitch, yaw).
+An arbitrary motion is specified via a CSV file (`BaseMotionFileName`) which contains 19 columns: time, 3 translations (global), three successive rotations (global), 3 translation velocities, 3 rotational velocities (omega, in global), 3 translational accelerations and 3 rotational accelerations (alpha, in global). Example of arbitrary input files are given in :numref:`ad_inputfiles_examples`.
+
+
+An example of inputs for a sinusoidal surge motion is given below:
+
+.. code::
+
+    ----- Turbine(1) Motion [used only when AnalysisType=1] --------------------------
+    1         BaseMotionType(1)      - Type of motion prescribed for this base {0: fixed, 1: Sinusoidal motion, 2: arbitrary motion} (flag)
+    1         DegreeOfFreedom(1)     - {1:xg, 2:yg, 3:zg, 4:theta_xg, 5:theta_yg, 6:theta_zg} [used only when BaseMotionType=1] (flag)
+    5.0       Amplitude(1)           - Amplitude of sinusoidal motion   [used only when BaseMotionType=1]
+    0.1       Frequency(1)           - Frequency of sinusoidal motion   [used only when BaseMotionType=1]
+    "unused"  BaseMotionFileName(1)  - Filename containing arbitrary base motion (19 columns: Time, x, y, z, theta_x, ..., alpha_z)  [used only when BaseMotionType=2]
+
+
+The different inputs for the basic and advanced geometries are given below:
+
+- basic: The motion of a basic turbine consist of a constant nacelle yaw (`NacYaw`), rotor speed (`RotSpeed`), blade pitch (`BldPitch`). 
+  Examples are given below:
+
+.. code::
+
+    0         NacYaw(1)        - Yaw angle (about z_t) of the nacelle (deg)
+    7         RotSpeed(1)      - Rotational speed of rotor in rotor coordinates (rpm)
+    1         BldPitch(1)      - Blades pitch (deg)
+
+- advanced: When an advanced geometry is provided and when the number of blades is non-zero, the motion section contains options for the nacelle motion, rotor motion and individual blade pitch motion. 
+  The syntax for each of these motion consists of defining a type (fixed or time-varying), a value for the fixed case or a file for the time-varying case.
+  The input files are CSV files containing time, position, speed and acceleration. Examples of files are given in :numref:`ad_inputfiles_examples`.
+  If the pitch rate, yaw rate, and accelerations are not known, the user should input zeros in these columns.
```

Comment:
I'm not sure this needs to be stated (may cause confusion).  Instead, I would say--as above--"The displacements/orientations, velocities, and accelerations are not checked internally for consistency."